### PR TITLE
refactor: merge worker into fresh

### PR DESCRIPTION
**Pull request by crowlKats**
Questions:
- [x] which responses do we set `Access-Control-Allow-Origin: *`? all of them?
- [x] what to do with caching?
- [x] What do we do with the current `proxyFile` function?

### CODE REVIEW

File: worker/main.ts
Comment (ry) [inline worker/main.ts:8]: `//worker/main.ts` should just be a symlink to `//main.ts` ?

Comment (crowlKats) [reply worker/main.ts:8]: yes, this is just old code to keep reference with while porting

File: deps.ts
Comment (ry) [inline deps.ts:1]: This is getting a little ridiculous - we need to have a normal review & release process for code going into fresh if we're going to depend on it for our core infrastructure. 

cc @lucacasonato @piscisaureus

**Review by ry — COMMENTED** [review]
What happens when this PR lands. The dotland.deno.dev still has its entrypoint as `worker/main.ts` but that file is a symlink here? I thought the symlinks where not working?

File: deps.ts
Comment (crowlKats) [reply deps.ts:1]: we moved to bert's fork since luca isnt around to review & merge PRs

**Comment by crowlKats** [comment]
@ry i just committed a workaround

**Comment by ry** [comment]
@crowlKats [The preview deployment](https://dash.deno.com/projects/dotland/deployments/k4016qskg72g) seems broken. Getting this error:
```
TypeError: Must be a file URL.     at fromFileUrl (https://deno.land/std@0.108.0/path/posix.ts:408:15)     at Function.fromRoutes (https://raw.githubusercontent.com/piscisaureus/fresh/d990a0bd6b76b2c2ec4016e9bf2584c7c5c8fe88/src/server/context.ts:120:48)     at https://raw.githubusercontent.com/denoland/dotland/a7f1cd25da1cc471cb4d54d3210e91abadc69478/main.ts:65:33
```

**Comment by crowlKats** [comment]
It really doesn't help that the source mapping is completely fucked up and debugging this is quite annoying as the reported lines dont make sense

File: completions_v2.ts
Comment (ry) [inline completions_v2.ts:1]: Is this module new code? I can't tell where it came from?

Comment (crowlKats) [reply completions_v2.ts:1]: This is https://github.com/denoland/dotland/blob/main/worker/suggestions.ts, but went through two refactors

File: server_deps.ts
Comment (ry) [inline server_deps.ts:1]: Please use https://deno.land/x/deno_fresh@0.0.1/server.ts

File: deps.ts
Comment (ry) [reply deps.ts:1]: Use https://raw.githubusercontent.com/denoland/fresh/0.0.2/runtime.ts

This doesn't include bert's etag patch - we should fix https://github.com/denoland/deploy/issues/2758 before landing this one.

File: completions_v2.ts
Comment (lucacasonato) [inline completions_v2.ts:1]: Why is this file at the root of the tree?

**Review by lucacasonato — COMMENTED** [review]
LGTM, if you have your finger ready on the revert button :-)

**Review by lucacasonato — APPROVED** [review]

File: completions_v2.ts
Comment (crowlKats) [reply completions_v2.ts:1]: well, it isnt part of pages, static, utils, nor tests, so it has no place to belong, + this is being imported directly by main.ts

**Pull request merged**
